app/services/api/v1x1/tags.rb

```diff
@@ -0,0 +1,7 @@
+module Api
+  module V1x1
+    module Tags
+
```

Comment:
Same here for `class CollectLocalOrderResources < Api::V1x0::Tags::CollectLocalOrderResources; end`, right?

Do we need `class Topology::RemoteInventory < Api::V1x0::Tags::Topology::RemoteInventory; end` too?

Reply:
Same as https://github.com/RedHatInsights/catalog-api/pull/648#discussion_r410467046